app/assets/javascripts/views/cost_packages.js

```diff
@@ -59,7 +59,7 @@
           axis: {
             x: {
               type: 'category',
-              categories: _.uniq(_.pluck(data, 'year')),
+              categories: [2016, 17, 18, 19, 20, 21, 22, 23, 24, 25],
```

Comment:
This is ok for me but I'm not sure if the data starts at 2015 or 2016. Can you check?